packaging/android/build.sh

```diff
@@ -325,10 +348,25 @@ if [ -e "$PREFIX/lib/libftdi1.so" ] ; then
 	rm "$PREFIX"/lib/libftdi1.so*
 fi
 
+# build libdivecomputer
+
+if [ ! -d libdivecomputer ] ; then
```

Comment:
I don't like this. This will end up cloning libdivecomputer into the build directory, and not next to the subsurface source which is what we're building from.

We have the (now a bit outdated) packaging/android/android-build-wrapper.sh for orchestrating those parts.